holoviews/core/io.py

```diff
@@ -854,3 +854,10 @@ def contents(self, maxlen=70):
     def listing(self):
         "Return a list of filename entries currently in the archive"
         return ['.'.join([f,ext]) if ext else f for (f,ext) in self._files.keys()]
+
+    def clear(self):
+        "Clears the file archive"
+        return self._files.clear()
```

Comment:
Don't think `dict.clear()` returns anything

```suggestion
        self._files.clear()
```